src/components/lib/posix/posix.c

```diff
@@ -221,16 +226,20 @@ cos_nanosleep(const struct timespec *req, struct timespec *rem)
 		return -1;
 	}
 
-	wakeup_deadline        = sl_now() + sl_usec2cyc(time_to_microsec(req));
-	completed_successfully = sl_thd_block_timeout(0, wakeup_deadline);
-	wakeup_time   = sl_now();
+	//wakeup_deadline        = sl_now() + sl_usec2cyc(time_to_microsec(req));
+	wakeup_deadline = 0;
```

Comment:
I don't think that silently failing is the best policy here. We should probably fail loudly if a function that we don't support is called. BUG(), assert(0), after a printc to explain. etc...

I don't trust client checking for `ENOTSUP` which would be the normal solution. Failure is more appropriate if it isn't supported here.